### DISCUSSION TITLE
Migration script in v3

### DISCUSSION THREAD

**Discussion by quirm** [sanity-io/sanity#3818] [Sanity Studio Support] [answered]
Hi, what package would give me a client for a migration script in v3? e.g. this v2 script: https://www.sanity.io/docs/migrating-data#0f6618dcaacf ?

**Comment by safejace** [comment]
would love to know this too

**Comment by rexxars** [comment, marked as answer]
Hi,

You should be able to run the same migration scripts with a few minor tweaks:

In v2:
```ts
import sanityClient from 'part:@sanity/base/client'

const client = sanityClient.withConfig({ apiVersion: '2021-08-21' })
```

In v3:
```ts
import {getCliClient} from 'sanity/cli'

const client = getCliClient({ apiVersion: '2021-08-21' })
```
